src/app/caesar-portal/common/pipes/mathces-progress.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Group } from '../models/group'
+import { GroupStatus } from '../models/group-status'
+
+@Pipe({
+  name: 'mathcesProgress'
+})
+export class MathcesProgressPipe implements PipeTransform {
+  transform(source: Group[], status: GroupStatus): Group[] {
+    if (source) {
+      return source.filter((item: Group) => {
```

Comment:
in arrow functions you can write (item: Group) => item.status === status and it will create block with return